src/modules/Dashboard/features/RespondentData/RespondentDataReview/RespondentDataReview.tsx

```diff
@@ -233,7 +233,9 @@ export const RespondentDataReview = () => {
 
     const lastActivityCompletedDate = lastActivityCompleted && new Date(lastActivityCompleted);
     const selectedDateInParam =
-      selectedDateParam && isValid(new Date(selectedDateParam)) && new Date(selectedDateParam);
+      selectedDateParam &&
```

Comment:
This is addressing the date parsing issue while inconvenient and nice to finally have a fix for, is not the root cause of the issue in this ticket.

The primary issue is the calendar is returning dates for other participants which when selected, don't contain any responses making it hard to determine which dates need to be reviewed.

https://github.com/user-attachments/assets/7e90f097-272c-4fba-8df2-6dfcdb701f9c

Reply:
@ChaconC I saw that this was only a partial fix, too, and it's an important part of it, because the date offset was actually preventing you from accessing certain dates that you should be able to access for the current user. I guess that to complete the fix (and before this can be be sent over to QA), the [separate BE ticket](https://mindlogger.atlassian.net/browse/M2-8215) also needs to be fixed. Is that right? Or are there more FE changes needed?

Reply:
That's right @farmerpaul! Both parts need to be completed to close this one out